<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QFrame" name="frame_left_panel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="grp_visualization">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>可视化控制区</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radio_orient_axial">
        <property name="text">
         <string>Axial</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_orient_sagittal">
        <property name="text">
         <string>Sagittal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_orient_coronal">
        <property name="text">
         <string>Coronal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_registration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>配准参数区</string>
    </property>
    <widget class="QLabel" name="lbl_reg_mode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>45</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>配准模式：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_reg_mode">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Rigid (刚性)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Affine (仿射)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B-Spline（B样条）</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lbl_coarse_mode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>粗配准：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_coarse_mode">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>基于几何中心</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>重心对齐</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>骨骼特征匹配</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>数据加载区</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_load_fixed">
        <property name="text">
         <string>加载参考图像 (Fixed)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_load_moving">
        <property name="text">
         <string>加载移动图像 (Moving)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_actions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>操作区</string>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>Axial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="text">
         <string>Sagittal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_6">
        <property name="text">
         <string>Coronal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_start_stitching">
        <property name="text">
         <string>开始自动拼接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_export_result">
        <property name="text">
         <string>导出结果</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_fusion_controls">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>不透明度控制</string>
    </property>
    <widget class="QCheckBox" name="chk_check_alignment">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>检查骨骼对齐</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="lbl_legend_fixed">
        <property name="text">
         <string>Fixed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="scroll_opacity">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_legend_moving">
        <property name="text">
         <string>Moving</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_views">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>481</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>视图区</string>
   </property>
   <widget class="QLabel" name="lbl_title_difference">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Difference Map</string>
    </property>
   </widget>
   <widget class="QVTKOpenGLNativeWidget" name="view_fixed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title_fusion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fusion Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title_moving">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Moving Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title_fixed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixed Image</string>
    </property>
   </widget>
   <widget class="QVTKOpenGLNativeWidget" name="view_moving">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKOpenGLNativeWidget" name="view_fusion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKOpenGLNativeWidget" name="view_difference">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_status_panel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>状态区</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lbl_status_message">
       <property name="text">
        <string>正在计算互信息...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progress_bar_main">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
